inventory/vagrant/group_vars/webserver/php-RedHat.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+php_date_timezone: "America/Toronto"
```

Comment:
This variable is the same as in `php-Debian.yml` so I'd pull it out into a separate variable file.

Reply:
Actually, I would be fine simply removing it. The default from geerlingguy is Chicago, but I'm in Las Vegas, so setting a default for Toronto (although consistent w/ Islandora's origin) doesn't really make sense to me. Might as well set it to GMT.

Reply:
@seth-shaw-unlv Good point @dannylamb @Natkeeran @jonathangreen what are your feelings on this, do we really need to set the datetime zone?

Reply:
Sometimes things complain if it's not set.  I'd just make it configurable with a default of GMT.

Reply:
We discussed this at the CLAW call and the consensus is that as long as it is set (and it is set to [Chicago](https://github.com/geerlingguy/ansible-role-php/blob/462ebfaabf7ec41f609c35dae43c3d34cddf6760/defaults/main.yml#L63) as @seth-shaw-unlv said) then we can remove these